assets/src/styles/blocks/core-overrides/BlockSpacing.scss

```diff
@@ -23,6 +22,20 @@
   }
 }
 
+.page-content > .wp-block-media-text {
```

Comment:
Maybe we should also apply this change to `.page-content > .wp-block-group`? A lot of our patterns and pattern layouts are within a Group block 🤔

Reply:
Shouldn't it be just `.page-content > .wp-block-group`?

Reply:
I applied [this code](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1108/files#diff-1791ed999879035510ea6eb81b4e9f3997399ef07c9a3919594d80ed35dd78e2R25) to fix cases like this [demo page](https://www-dev.greenpeace.org/test-neptune/3100-2/) and avoid having double empty space.

But I think if we apply `.page-content > .wp-block-group`, we will lose all the empty space between some blocks into pages, for example into a layout patterns.

Reply:
Also the new [fix](https://www-dev.greenpeace.org/test-neptune/3100-2/) fit the [UI](https://www.figma.com/file/qmQBzfSlU1TgsUPT37M0eX/Page-pattern-layouts_new?type=design&node-id=10-831&mode=dev)

Reply:
Actually, this is only an issue when there's no page title, right? Maybe we should add these overrides [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/pages/_page.scss#L12) then?

Reply:
@dantovbein what do you think about my last comment?

Reply:
Hey @mleray Here is [another PR ](https://github.com/greenpeace/planet4-master-theme/pull/2147)